pkg/kubectl/cmd/run.go

```diff
@@ -562,7 +623,11 @@ func generateService(f cmdutil.Factory, cmd *cobra.Command, args []string, servi
 	return nil
 }
 
-func createGeneratedObject(f cmdutil.Factory, cmd *cobra.Command, generator kubectl.Generator, names []kubectl.GeneratorParam, params map[string]interface{}, overrides, namespace string) (runtime.Object, string, meta.RESTMapper, *meta.RESTMapping, error) {
+func createGeneratedObject(f cmdutil.Factory, cmd *cobra.Command,
```

Comment:
this change is only for keep line length reasonable right ? do we have a rules for that ?  gofmt golint check ?, cause if we gonna follow some rules, there's lot of code right now keeping unreasonable line length.